content/release-10-17-0/streaming-analytics-10-17-0-bundle/10_17_0.md

```diff
@@ -73,3 +73,7 @@ The **Latest Updated** mode of the
 [Combiner](https://documentation.softwareag.com/pab/10.17.0/en/webhelp/pab-webhelp/index.html#page/pab-webhelp%2Fre_AnaBui_block_reference_Flow_Manipulation_Combiner.html) block, which has been added in
 [release 10.16.0](/release-10-16-0/streaming-analytics-10-16-0/),
 has been renamed to **Latest Input**. This was done so that you can better distinguish it from the **Latest Changed** mode.
+
+### Changes to microservice health endpoint
```

Comment:
I think this info should be moved into the "Important announcements" in content/release-10-17-0/announcements-10-17-0.md

Reply:
Good point, it fits with everything else we put there, even though it's minor.

20cb144